src/game.cpp

```diff
@@ -2722,19 +2722,35 @@ void Game::playerAcceptTrade(uint32_t playerId)
 		}
 
 		bool isSuccess = false;
+		
+		ReturnValue tradePartnerRet = RETURNVALUE_NOERROR;
+		ReturnValue playerRet = RETURNVALUE_NOERROR;
+
+		// if player is trying to trade its own backpack
+		if (tradePartner->getInventoryItem(CONST_SLOT_BACKPACK) == partnerTradeItem) {
+			slots_t playerSlotItem = getSlotType(Item::items[playerTradeItem->getID()]);
```

Comment:
as far as I see this variable and the other one are used only once right? If you aren't checking them or anything I think you can just put the code as 
```
tradePartnerRet = (tradePartner->getInventoryItem(getSlotType(Item::items[playerTradeItem->getID()])) ? RETURNVALUE_NOTENOUGHROOM : RETURNVALUE_NOERROR);
```
What do you think?